helm/v1beta1/types_project_helm_chart_repository.go

```diff
@@ -19,13 +20,68 @@ type ProjectHelmChartRepository struct {
 	// spec holds user settable values for configuration
 	// +kubebuilder:validation:Required
 	// +required
-	Spec HelmChartRepositorySpec `json:"spec"`
+	Spec ProjectHelmChartRepositorySpec `json:"spec"`
 
 	// Observed status of the repository within the namespace..
 	// +optional
 	Status HelmChartRepositoryStatus `json:"status"`
 }
 
+// Project Helm chart repository exposed within a namespace
+type ProjectHelmChartRepositorySpec struct {
```

Comment:
Was this not done on TLS PR already, would only expect a new change for the new BasicAuth field.